docs/tutorials/new-feature-tutorial.md

```diff
@@ -293,9 +293,9 @@ active users in a realm.
 
     # zerver/lib/actions.py
 
-    def do_set_realm_property(realm: Realm, name: str, value: bool) -> None:
-      """Takes in a realm object, the name of an attribute to update, and the
-      value to update.
+    def do_set_realm_property(realm: Realm, name: str, value: bool, acting_user: Optional[UserProfile]=None) -> None:
+      """Takes in a realm object, the name of an attribute to update, the value to update and an the UserProfile
+      which initiated the update.
```

Comment:
I fixed this, but FYI the text here didn't match the `actions.py` version and had some typos.